map.json

```diff
@@ -33,5 +33,10 @@
 		"id" : "win-bcdedit-ignoreAllFailure",
 		"path" : "src/windows/win-ignoreAllFailures.ps1",
 		"description" : "Ignore errors if there is a failed boot, failed shutdown, or failed checkpoint. The computer will attempt to boot normally after an error occurs."
+	},
+	{
+		"id" : "linux-recover",
```

Comment:
changed the id to 'alar-fki' This stand for 'azure-linux-auto-recover' 'fki' reflects the functionality
fstab
kernel
initrd
So in case we have another scenario added in the future we can use a name line this one 'alar-passwd' to recover a password issue for instance.

In order to reflect the new name I have also changed the folder structure
Inside the help folder we have the alar folder and inside the alar-fki. In this alar-fki you will find all the other scripts